docs(mpc): update readme for how-to-tune-mpc (#3981)
autowarefoundation/autoware.universe: control/mpc_lateral_controller/README.md

- `weight_terminal_heading_error`: Preferable to set a higher value than normal heading weight `weight_heading_error`
  for stability.

#### Other tips for tuning

- In theory, increasing terminal weights, `weight_terminal_lat_error` and `weight_terminal_heading_error`, can enhance the tracking stability. This method sometimes proves effective.

> - A larger `prediction_horizon` and a smaller `prediction_sampling_time` are efficient for tracking performance. However, these come with a trade-off between computational costs.

Reviewer: ```suggestion
- A larger `prediction_horizon` and a smaller `prediction_sampling_time` are efficient for tracking performance. However, these come at the cost of higher computational costs.
```
